pyproject.toml

```diff
@@ -10,7 +10,7 @@ build-backend = "setuptools.build_meta"
 [tool.cibuildwheel]
 skip = "pp* *musllinux*"
 test-skip = "*i686 *win32 *-macosx_arm64"
-before-test = "pip install cython; pip install --extra-index-url https://test.pypi.org/simple energyflow==1.3.3a0"
+before-test = "pip install cython; pip install git+https://github.com/thaler-lab/EnergyFlow.git"
```

Comment:
This is simplified compared to the contents of `.github/workflows/ci.yml`, as the way that `cibuildwheel` interacts with Python 3.7 on Windows causes issues otherwise.